MediaAdminBundle/Resources/public/coffee/GalleryLoad.coffee

```diff
@@ -1,4 +1,5 @@
-GalleryLoad = (link, target) ->
+GalleryLoad = (link, target, mediaView) ->
+  mediaView = mediaView or "showGalleryCollection";
```

Comment:
After a second read, I'll change the parameters order and add a coffee default value to make something like this :
GalleryLoad = (link, mediaView = "showGalleryCollection", target) ->
But beware to report the modification on GalleryLoad call